iocore/net/SSLNetVConnection.cc

```diff
@@ -455,7 +454,6 @@ SSLNetVConnection::read_raw_data()
 
     if (this->has_proxy_protocol(buffer, &r)) {
       Debug("proxyprotocol", "ssl has proxy protocol header");
-      set_remote_addr(get_proxy_protocol_src_addr());
```

Comment:
This is the main change.

Reply:
I'm wondering if we can unify these code in the `SSLNetVConnection` and `ProtocolProbeSessionAccept`, but it's out of scope from this PR.